### ISSUE TITLE
Settings toggle to universally disable composer markdown

### ISSUE THREAD

**Issue by ShadowJonathan** [vector-im/element-web#20321]
#### What would you like to do?

Disable markdown on the composer for a user.

#### Why would you like to do it?

Some users in https://github.com/vector-im/element-web/issues/483 would like it, as prepending `/plain` to every message is not feasable for them.

#### How would you like to achieve it?

Add a setting under Settings -> Preferences -> Composer

### Additional context

This is a breakout of https://github.com/vector-im/element-web/issues/483, this is a separate/specific issue, as that issue collates both popping up the UI and disabling the setting into one, while i believe this is better actionable from a Design standpoint.

**Comment by aaronraimist** [comment]
I don't understand how this is different from #483

**Comment by ShadowJonathan** [comment]
That issue is bikeshedding over the warning + in-composer toggle, im mainly going from comments like https://github.com/vector-im/element-web/issues/483#issuecomment-596886396 and https://github.com/vector-im/element-web/issues/483#issuecomment-716231528, which ask to break this out into a settings toggle.

**Comment by dreaddr** [comment]
Please just implement an option to disable markdown for gosh sake, I can't believe its taken more than **6 years** now.

**Comment by bradyt** [comment]
Another way to phrase it might be, prioritize and fast-track a feature that prevents the need to write `/plain` before every message.

**Comment by bradyt** [comment]
TL;DR, underscores are silently dropped.

Another gotcha moment. User wants to send underscores as text. Since some markdown makes it through, they might expect all markdown makes it. `> a quote` makes it through, despite Element converting to html quote, the IRC bridge converts it back to markdown, so it is preserved. On the other hand, `_some underscored text_` is converted by element to italic html, IRC bridge converts it to control codes `^]some underscored text^O`, and some Libera.chat channels have set `+c` which strips control code formatting, so underscores are not preserved. This is one of several non-obvious issues that can take time to understand what is going on, and would be avoided if users could just have something like `/plain` as a default setting.

**Comment by ostwilkens** [comment]
The markdown support causes more trouble than it's worth. I'm chatting, not writing a readme or blogpost.

**Comment by niquewoodhouse** [comment]
> Add a setting under Settings -> Preferences -> Composer

I think that makes a lot of sense.

<img width="1440" alt="Settings-md" src="https://user-images.githubusercontent.com/1587166/163358426-46e09b71-b348-44cf-9f86-703a27c1d541.png">

Just adding a bit of copy to let people know, for now, how they can disabled/enable temporarily for specific messages.   I don't have enough insights on whether or not md should be off by default so I'd suggest for now leaving the default behaviour as it is and creating separate issues for that if needed later.

**Comment by ShadowJonathan** [comment]
Nit/bikeshed; I think the subtext should be;
> Override with the `/plain` command to send with markdown disabled.
...when the feature is enabled, and;
> Override with the `/md` command to send with markdown enabled.
...when it's not.
